client/src/components/ErrorMessageRecs.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import styled from '@emotion/styled';
+
+const Container = styled.div`
+  width: 80%;
+  border: 1px solid #aeb2f5;
+  text-align: center;
+  padding: 15px 10px;
+  text-shadow: 0px 1px 4px #f79dc1;
+  box-shadow: 1px 1px 3px #aeb2f5;
+  font: 200 0.9rem 'Roboto', sans-serif;
+  color: rgb(255, 255, 255);
+  margin: 30px;
+  line-height: 18px;
+  letter-spacing: 1px;
+
+  a {
+    color: #aeb2f5;
+    text-decoration: underline;
+    font-style: italic;
+    text-shadow: none;
+
+    :visited {
+      color: rgb(231, 234, 255, 0.9);
+      text-decoration: underline;
+      font-style: italic;
+    }
```

Comment:
Are you sure about this `visited` styling? It's very uncommon in modern web development.

Reply:
The default visited styling is a purple colour that is very hard to see with my colour theme so I changed it...